app/utils/Regex.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import scala.util.matching.Regex
+
+object Regex {
+
+  val UtrRegex: Regex = "^([0-9]{10})$".r
+  val UrnRegex: Regex = "^((?i)[a-z]{2}trust[0-9]{8})$".r
```

Comment:
Bit picky but can these should probably be camel case

Reply:
@jackheslop96 I followed convention which is to start regex pattern matching with an Uppercase. Otherwise I think the compiler doesn't know about the unapply() method.